.github/CODEOWNERS

```diff
@@ -457,7 +457,7 @@
 /core/domain/email*.py @aks681
 /core/domain/image_service*.py @DubeySandeep
 /core/domain/job_validators*.py @vojtechjelinek
-/core/domain/prod_validators*.py @vojtechjelinek
+/core/domain/subtopic_validators*.py @vojtechjelinek
```

Comment:
Move this to `New structure project` section*

Reply:
Done